src/redux/store/ItemsReduce.js

```diff
@@ -1,8 +1,16 @@
 import imf from '../../img/bag_rose.jpg'
```

Comment:
a lot of empty lines. Remove it and lent only one. Don't need multiple empty lines